index.bs

```diff
@@ -10300,14 +10300,20 @@ Constructors</h5>
 				Throw a {{TypeError}} exception if the slot is
 				empty.
 
-			1. If any of following steps throws any exception,
-				abort the rest of the steps and
-				<a>queue a task</a> to the <a>control thread</a>
-				to <a spec="dom" lt="fire an event">fire an
-				event</a> named <code>processorerror</code>
-				using
-				<a href="https://www.w3.org/TR/html50/webappapis.html#the-errorevent-interface">ErrorEvent</a>
-				at <var>nodeReference</var>.
+			1. If any of following steps throws any exception:
```

Comment:
Phrasing here is a bit awkward.  Perhaps it would sound better if you said:
> If any of following steps throws any exception, perform these steps to handle the exception

Can't think of a good way to phrase this because "following steps" is a bit ambiguous. Is "following steps" the new steps you've added as a sub step of this one, or is it the other steps in this algorithm at the same level as this item?

Reply:
I rephrased a bit. PTAL again.